README.md

```diff
@@ -221,6 +246,13 @@ won't help you, and you'll need to write a lower-level CMake
 file. This README can't cover all the possible situations here, but
 the `nowide` and `catch` CMake files are both solid examples.
 
+## How To Release
+
+1. Update [CHANGELOG.md](CHANGELOG.md) with release notes based on
+``git log `git describe --abbrev=0 --tags`..HEAD``
+1. Update the version in the project declaration of [CMakeLists.txt](CMakeLists.txt)
```

Comment:
That doesn't seem like something we'd do around releasing. That seems more like something you do when using Leatherman, and you'd do it by specifying `find_package(Leatherman 0.0.6)`.